cmd/earthly/flags.go

```diff
@@ -75,7 +76,15 @@ func (app *earthlyApp) rootFlags() []cli.Flag {
 			Value:       "https://api.earthly.dev",
 			EnvVars:     []string{"EARTHLY_SERVER_ADDRESS"},
 			Usage:       "API server override for dev purposes",
-			Destination: &app.apiServer,
+			Destination: &app.cloudHTTPAddr,
+			Hidden:      true, // Internal.
+		},
+		&cli.StringFlag{
+			Name:        "grpc",
+			Value:       "ci.earthly.dev:443",
```

Comment:
Could we make this the same format as the address above? I know it's internal, but it would help with our own internal consistency.

`https://ci.earthly.dev` (the port would be implied by `https`)

Reply:
I don't think gRPC likes the protocol being set. It likes the format to be `host:port`. I get an error like this if I try to set it to `https://ci.earthly.dev`. 
```
transport: Error while dialing dial tcp: lookup tcp///ci.earthly.dev: nodename nor servname provided, or not known
```

Otherwise, I would have just re-used the existing server address we have for the other HTTP calls 😞

Reply:
Yeah, what I meant was for us to take the `https://ci.earthly.dev` address and turn it into `ci.earthly.dev:443` automatically underneath before passing it to gRPC.

Reply:
Right - I actually had it doing something similar at first but then changed it. 

I imagined devs pointing earthly to their remote or local dev environment, and that they might get blocked if the parsing rules I wrote weren't flexible enough for their use case. Leaving the full address configurable is less code and probably more convenient.  Plus eventually we'll prob drop the HTTP address and only have a single one for gRPC anyways.